templates/zerver/stream_creation_prompt.html

```diff
@@ -7,9 +7,9 @@ <h3 id="stream-creation-label">{% trans %}Create stream{% endtrans %}</h3>
   <form id="stream_creation_form" class="form-inline">
       <div class="modal-body">
           <div>
-            <b>{% blocktrans %}Stream name{% endblocktrans %}</b><br />
+            <b>{% trans %}Stream name{% endtrans %}</b><br />
             <input type="text" name="stream_name" id="create_stream_name"
-                 placeholder="Stream name" value="" />
+                 placeholder="{{ _('Stream name') }}" value="" />
```

Comment:
you should move these fixes to the first commit into the first commit; otherwise the first commit will totally fail tests when we're bisecting.